dist/button/button.css

```diff
@@ -171,6 +171,12 @@ button.btn--primary {
         var(--color-foreground-on-accent)
     );
     font-weight: 700;
+    transition: all var(--motion-duration-short-3)
+        var(--motion-easing-quick-enter);
+}
+a.fake-btn--primary:active,
```

Comment:
According to the issue, motion is supposed to be applied only to,

`CTA primary, CTA secondary, CTA tertiary buttons, and Branded buttons.`

This is applying it to all buttons with a priority, and the CTA button doesn't appear to have it applied.

Reply:
So cta buttons for design are these buttons. 
Our cta buttons are different.

Reply:
Oh wow! Ok.